docker-compose.yml

```diff
@@ -5,14 +5,13 @@ services:
     image: dashpay/dashd:${CORE_VERSION:-latest}
     restart: always
     ports:
-      - ${CORE_P2P_PORT}:20001 # P2P
+      - ${CORE_P2P_PORT}:${CORE_P2P_PORT} # P2P
```

Comment:
We shouldn't start if var is empty. Please mark it as required: `${CORE_P2P_PORT:?err}`

Reply:
I would use `${CORE_P2P_PORT:?err}` here too for consistency